src/client/components/friendsView/FriendNameInputPage.tsx

```diff
@@ -52,14 +52,14 @@ export default function FriendNameInputPage({}: FriendNameInputPageProps): JSX.E
   }
 
   const handleClick = () => {
-    history.push('/Menu')
+    history.push('/Menu/Friends')
   }
 
   return (
     <ContainerDiv>
 
       <OrangeNavbar/>
-      <HeaderImage src='Dannys_bg.png'/>
+      <HeaderImage src='/Dannys_bg.png'/>
```

Comment:
Ohhhhh think i get what you meant with Header Image

Reply:
Yeah, I gotta save this lol